mlrun/runtimes/databricks/databricks_wrapper.py

```diff
@@ -84,10 +86,21 @@ def print_status(run: Run):
             ],
         )
         logger.info(f"starting to poll: {waiter.run_id}")
-        run = waiter.result(
-            timeout=datetime.timedelta(minutes=timeout_minutes),
-            callback=print_status,
-        )
+        try:
+            run = waiter.result(
+                timeout=datetime.timedelta(minutes=timeout_minutes),
+                callback=print_status,
+            )
+        except OperationFailed:
+            # TODO handle rerun tasks - so we can not take the first task in tasks list.
```

Comment:
Why not deal with it now?

Reply:
I will open another PR to handle rerun issue:)

Reply:
Why? This is new code. Might as well get it right the first time if there's no particular reason not to.